src/main/target/ANYFCF7/target.h

```diff
@@ -24,6 +24,7 @@
 #define USBD_PRODUCT_STRING "AnyFCF7"
 
 #define USE_DSHOT
+#define USE_ESC_TELEMETRY
```

Comment:
I would agree if this could be done in `common.h`, but doing it in each individual `target.h` makes the target files harder to read and maintain.